code/modules/clothing/head/collectable.dm

```diff
@@ -100,9 +102,10 @@
 
 /obj/item/clothing/head/collectable/kitty
 	name = "collectable kitty ears"
-	desc = "The fur feels.....a bit too realistic."
+	desc = "The fur feels... a bit too realistic."
 	icon_state = "kitty"
 	body_parts_covered = 0
+	style = STYLE_NEG_HIGH//no.
```

Comment:
Bruh. It's not less stylish than other hats and also unique.

Reply:
this is a cat ear headband/tail thing iirc, not a hat.
Also, Eris does not like felinids.

Reply:
Imo leaving it at 0 is a disadvantage enough. You miss out on style, and get dead by three headshots.

Reply:
You can't stop me from wearing them